docs/reference/query-dsl/multi-match-query.asciidoc

```diff
@@ -516,3 +519,55 @@ per-term `blended` queries. It accepts:
 ===================================================
 The `fuzziness` parameter cannot be used with the `cross_fields` type.
 ===================================================
+
+[[type-bool-prefix]]
+==== `bool_prefix`
+
+The `bool_prefix` type's scoring behaves like <<type-best-fields>>, but using a
+<<query-dsl-bool-prefix-query,`bool_prefix` query>> instead of a `match` query.
+
+For example, the query
+
+[source,js]
+--------------------------------------------------
+GET /_search
+{
+  "query": {
+    "multi_match" : {
+      "query":      "quick brown f",
+      "type":       "bool_prefix",
+      "fields":     [ "subject", "message" ]
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+
+would be roughly equivalent to the query
```

Comment:
There is an example for `most_fields` already so let's not repeat here if we fix the previous link ?

Reply:
That makes sense, I'll remove this example